src/poker/game/HandsAndBoard.java

```diff
@@ -0,0 +1,33 @@
+package poker.game;
+
+import poker.Card;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HandsAndBoard {
+    private List<Card> hand1;
+    private List<Card> hand2;
+    private List<Card> board;
+
+    public HandsAndBoard(List<Card> hand1, List<Card> hand2, List<Card> board){
+        this.hand1 = hand1;
+        this.hand2 = hand2;
+        this.board = board;
+    }
+    public List<Card> getHand1(){
+        return hand1;
+    }
+    public List<Card> getFlop(){
+        return new ArrayList<>(board.subList(0, 3));
+    }
+    public List<Card> getTurn(){
+        return new ArrayList<>(board.subList(0, 4));
+    }
+    public List<Card> getHand2(){
+        return hand2;
+    }
+    public List<Card> getBoard(){
```

Comment:
troche czepialskie ale raczej jak zrobiles funckje getFlop() i getTurn() to to powinna byc funkcja getRiver()

Reply:
co z tym?